cms/static/sass/elements/_forms.scss

```diff
@@ -138,6 +138,100 @@ form {
   }
 }
 
+
+.license-img {
+  padding: 4px;
+}
+
+//ELEM: license selector wraper
+.license-selector {
+  text-align: middle;
+
+  .action.license-button {
+    @include grey-button;
+    @extend %t-action2;
+
+    display: inline-block;
+
+    text-align: center;
+
+    width: 220px;
+    height: 40px;
+
+    cursor: pointer;
+
+    &.selected {
+      @include blue-button;
+    }
+  }
+  .license-cc-options {
+    .license-button {
+      line-height: 1.5;
+      border-bottom: 1px solid #EEE;
+      padding: ($baseline / 2) 0 ($baseline * 0.4);
+      &.clickable {
```

Comment:
We use stateful classnames. So, `clickable` (if something is clickable), should be `is-clickable`. Related, if something _has_ a state we use `has-_state_`.